.github/workflows/ci.yml

```diff
@@ -0,0 +1,235 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  sanity:
+    name: Sanity
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.1.0
+
+      - name: Install Python
+        uses: actions/setup-python@v4.3.0
+        with:
+          python-version: '3.8'
+
+      - name: Run pre-commit checks
+        uses: pre-commit/action@v3.0.0
+        env:
+          SETUPTOOLS_USE_DISTUTILS: stdlib
+
+      - name: Check clowadd manifest
```

Comment:
nitpick: I think this is supposed to be clowdapp

Reply:
Yup. Thanks!